docs/modules/documentation/containers/modal/modal-docs.component.html

```diff
@@ -64,23 +56,65 @@ <h2>Component as Content</h2>
 
 <h2>Fullscreen Modal</h2>
 <description>
-    The modal dimensions can be controlled through the width and height inputs. These accept strings, so you may use whatever unit works best for your situation.
+    The modal dimensions can be controlled through the width and height properties of the configuration object. These
+    accept strings, so you may use whatever unit works best for your situation.
 </description>
 <component-example [name]="'ex4'">
-    <fd-modal-fullscreen></fd-modal-fullscreen>
+    <fd-modal-fullscreen-example></fd-modal-fullscreen-example>
 </component-example>
-<code-example [code]="fullScreenSource" [language]="'typescript'"></code-example>
+<code-example [code]="fullScreenSource" [language]="'html'"></code-example>
+<code-example [code]="fullScreenSourceTs" [language]="'typescript'"></code-example>
 
 <separator></separator>
 
-<h2>Modal in a Modal</h2>
+<h2>Stacked Modals</h2>
 <description>
-    <p>The modal component heavily relies on the ModalService, which contains every active Modal instance. To create nested modals, the service has an array of modal component references.</p>
-    <p>It treats this array like a stack, adding new modals to the top and calling pop() when they are closed. For this reason, nested modals must be closed in the order they were first opened.</p>
+    <p>The ModalService supports limitless stacked modals. Each can be closed individually, there is no specific closing
+        order.</p>
+    <p>In the example below, it is accomplished via components as content. However, it is entirely possible with
+        templates through the same methodology.</p>
 </description>
 <component-example [name]="'ex5'">
     <fd-modal-in-modal-example></fd-modal-in-modal-example>
 </component-example>
 <code-example [code]="modalInModalExample" [language]="'typescript'"></code-example>
 <code-example [code]="modalInModal" [language]="'typescript'"></code-example>
 <code-example [code]="modalInModalSecond" [language]="'typescript'"></code-example>
+
+<separator></separator>
+
+<h2>Backdrop Configuration</h2>
+<description>
+    The modal backdrop can be styled, disabled or even customized at will. Classes being applied to the backdrop must be
+    in the global styles file.
+</description>
+<component-example [name]="'ex6'">
+    <fd-backdrop-examples></fd-backdrop-examples>
+</component-example>
+<code-example [code]="backdropHtml" [language]="'html'"></code-example>
+<code-example [code]="backdropTs" [language]="'typescript'"></code-example>
+
+<separator></separator>
+
+<h2>ModalConfig</h2>
+<properties [properties]="{
+  properties: [
+    { name: 'id', description: 'String. The id of the modal element.' },
```

Comment:
similar to alerts, I find ID a bit odd for an api here

Reply:
I'm sortof projecting into the future, but I feel like at some point we'll want to get individual modals based on their id from the service. I dunno. Might as well have it, since it's otherwise impossible (or at least overly complicated) to set an id to the dynamically generated modal